src/pages/teams.astro

```diff
@@ -133,6 +134,29 @@ const future = false;
             and copy-pasting them to Livebook.</span
           >
         </div>
+        <div class="m-auto max-w-2xl flex flex-col items-center">
+          <img
+            class="sm:max-w-sm"
+            src={featureFileStorage}
+            alt="Feature: Synchronized file storage access"
+            width="571"
+            height="247"
+          />
+          <h2
+            class="text-gray-50 text-2xl md:text-3xl leading-tight mt-8 sm:mt-10"
+          >
+            Synchronized file storage access
+          </h2>
+          <span class="text-gray-200 mt-6 sm:mt-8"
+            >Effortlessly share and access files stored in cloud services like
+            Amazon S3 across your team, eliminating the need for manual
+            credential sharing. Livebook Teams ensures team members can run
+            notebooks relying on shared data without sacrificing security or
+            convenience. Moreover, Livebook Teams never gets access to your
+            storage credentials; they're encrypted to ensure that only your team
+            can use them.
```

Comment:
```suggestion
            >Effortlessly share notebooks and access data stored in cloud services like
            Amazon S3 across your team, eliminating the need for manual
            credential sharing. Moreover, Livebook Teams never gets access to your
            storage credentials; they're encrypted to ensure that only your team
            can use them.
```

I removed the sentence in the middle, as it was pretty much repeating the other ones (the first sentence is about convenience, the last one about security).